pkg/diag/validator/pod.go

```diff
@@ -235,3 +241,21 @@ func newPodStatus(n string, ns string, p string) *podStatus {
 func trimSpace(msg string) string {
 	return strings.Trim(msg, " ")
 }
+
+func getPodLogs(po *v1.Pod, c string) []string {
+	logCommand := []string{"kubectl", "logs", po.Name, "-n", po.Namespace, "-c", c}
+	logs, err := runCli(logCommand[0], logCommand[1:])
+	if err != nil {
+		return []string{fmt.Sprintf("Error retrieving logs for pod %s. Try `%s`", po.Name, strings.Join(logCommand, " "))}
+	}
+	lines := strings.Split(string(logs), "\n")
+	for i, s := range lines {
+		lines[i] = fmt.Sprintf("[%s %s]%s", po.Name, c, s)
```

Comment:
I think we should follow the skaffold logs format and have a space between the `]` and the message.